.goreleaser.yml

```
@@ -131,3 +131,54 @@ winget:
           owner: go-task
           name: winget-pkgs
           branch: "bump-task-to-{{.Tag}}"
+
+dockers:
```

`docker` would probably sound better 😄
```suggestion
docker:
```